plugins/pay.c

```diff
@@ -1733,7 +1733,11 @@ static void add_new_entry(struct json_stream *ret,
 			  const struct pay_mpp *pm)
 {
 	json_object_start(ret, NULL);
-	json_add_string(ret, "bolt11", pm->b11);
+	if(pm->b11){
+		json_add_string(ret, "bolt11", pm->b11);
+	}else{
+		json_add_sha256(ret, "payment_hash", pm->payment_hash);
```

Comment:
I think we should be printing the `payment_hash` irregardless of whether we have a `bolt11` string or not, they are about the only way to recognize a payment.

Reply:
I agree if we have the payment hash as result `listpays` is more simple to identify the payment inside the client app.

I will apport this change inside the next commit, and I remove the parenthesis to resolve also this review. https://github.com/ElementsProject/lightning/pull/3888#discussion_r462258558